sensor/stmemsc/README

```diff
@@ -56,3 +56,6 @@ License:
 
 License Link:
    https://opensource.org/licenses/BSD-3-Clause
+
+Patch List:
+   * sensor: lsm6dso: Disable -Wmaybe-uninitialized for lsm6dso_mode_set
```

Comment:
Nit: Add the impacted file, so we can quickly know which file to check: 
```suggestion
   * sensor: lsm6dso: Disable -Wmaybe-uninitialized for lsm6dso_mode_set
     Impacted file: sensor/stmemsc/lsm6dso_STdC/driver/lsm6dso_reg.c
```

Reply:
done